quodlibet/quodlibet/qltk/songsmenu.py

```diff
@@ -29,6 +29,17 @@
 from quodlibet.util.songwrapper import ListWrapper, check_wrapper_changed
 
 
+def confirm_song_removal_invoke(parent, songs):
+    songs = set(songs)
+    if len(songs) == 1:
+        title = _("Remove track: \"%s\" from library?"
+                  % next(iter(songs))['title'])
+    else:
+        title = _("Remove %s tracks from library?" % len(songs))
```

Comment:
This needs to use the `ngettext` (plural) form - I know you check for n=1, but languages behave quite differently with their pluralisation (e.g. some have several plural forms, and n=0 varies (though I guess this should never happen)).  

Also that should really be a `%d`...